lib/matplotlib/backends/backend_svg.py

```diff
@@ -96,24 +125,78 @@ def short_float_fmt(x):
     """
     return '{0:f}'.format(x).rstrip('0').rstrip('.')
 
-##
-# XML writer class.
-#
-# @param file A file or file-like object.  This object must implement
-#    a <b>write</b> method that takes an 8-bit string.
+
+def validate_xml(text):
+    """Validates that `text` is well-formed XML
+    
+    Parameters
+    ----------
+    text : str
+        XML text to validate
+    
+    Returns
+    -------
+    bool
+    
+    Raises
+    ------
+    ValueError
+    """
+    try:
+        ElementTree.fromstring(text)
+    except ElementTree.ParseError as e:
+        raise_from(ValueError("Invalid XML: %s" % e.msg), e)
+    return True
+
+
+def force_textual(obj):
+    """Convert `obj` into a `str` at all costs. Need
```

Comment:
I don't think this should be done. We should define the input as strings and if the user doesn't pass strings, it's on them.